containers/unit-tests/start

```diff
@@ -13,7 +13,7 @@ while test $# -gt 0; do
   case "$1" in
     --docker) docker="docker";;
     --podman) docker="podman";;
-    NO_NPM=1) no_npm='--env=NO_NPM=1';;
+    NO_NPM=1) no_npm='--env=NO_NPM=1';; # noop
```

Comment:
Why do we still need that then? Nothing looks at it any more.

Reply:
I left it there just in case it would get invoked from a workflow or something, but I guess not, actually...